rubicon/objc/types.py

```diff
@@ -568,16 +609,12 @@ def compound_value_for_sequence(seq, tp):
 
 register_preferred_encoding(b'^v', c_void_p)
 
-# Anonymous structs/unions with unknown fields can't be decoded meaningfully,
-# so we treat pointers to them like void pointers.
-register_encoding(b'^{?}', c_void_p)
-register_encoding(b'^(?)', c_void_p)
```

Comment:
Similarly, it's not clear why this is being removed...

Reply:
These are moved to a `with_encoding` annotation on `UnknownPointer`: https://github.com/beeware/rubicon-objc/pull/118/files#diff-a6b0ed51cad5a8aeefbf32642c5811adR648-R651

This is technically a functionality change that I snuck into the documentation PR, because while documenting I thought about this behavior, and decided that it makes more sense to map "unknown struct" to `UnknownPointer` rather than `c_void_p`.